app/src/main/res/layout/activity_confirm_request_leave.xml

```diff
@@ -30,309 +30,421 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/dp_5"
                     android:orientation="vertical"
                     >
-                    <TableLayout
+
+                    <LinearLayout
                         android:layout_width="match_parent"
-                        android:layout_height="match_parent"
-                        android:layout_margin="@dimen/dp_10"
-                        android:shrinkColumns="1"
-                        android:stretchColumns="1"
+                        android:layout_height="wrap_content"
+                        android:orientation="vertical"
                         >
-                        <TableRow
-                            android:gravity="center_vertical">
-                            <ImageView
-                                app:srcCompat="@drawable/ic_placeholder_user"
-                                tools:ignore="ContentDescription"
-                                style="@style/StyleIconConfirm"
-                                />
-                            <LinearLayout
-                                android:layout_marginLeft="@dimen/dp_20"
-                                android:layout_marginStart="@dimen/dp_20"
-                                android:orientation="vertical"
-                                >
-                                <TextView
-                                    android:text="@string/employee_name"
-                                    style="@style/StyleTitleConfirm"
-                                    />
-                                <TextView
-                                    style="@style/StyleTextConfirm"
+                        <TableLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="match_parent"
+                            android:layout_margin="@dimen/dp_10"
+                            android:shrinkColumns="1"
+                            android:stretchColumns="1"
+                            >
+                            <TableRow
+                                android:gravity="center_vertical">
+                                <ImageView
+                                    app:srcCompat="@drawable/ic_placeholder_user"
+                                    tools:ignore="ContentDescription"
+                                    style="@style/StyleIconConfirm"
                                     />
-                            </LinearLayout>
-                        </TableRow>
+                                <LinearLayout
+                                    android:layout_marginLeft="@dimen/dp_20"
+                                    android:layout_marginStart="@dimen/dp_20"
+                                    android:orientation="vertical"
+                                    >
+                                    <TextView
```

Comment:
WARNING: [TooDeepLayout] `activity_confirm_request_leave.xml` has more than 10 levels, bad for performance
 Layouts with too much nesting is bad for performance. Consider using a flatter layout (such as `RelativeLayout` or `GridLayout`).The default maximum depth is 10 but can be configured with the environment variable `ANDROID_LINT_MAX_DEPTH`.